app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<section class='signup'>
+  <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+    <nav class='nav'>
+      <div class='arrow' >
+        <%= link_to root_path, class: 'logo' do%>
+        <i class="fa-solid fa-arrow-left"></i>
+        <% end %>
+      </div>
+      <h2>REGISTER</h2>
+      <%= f.submit "Next" %>
+    </nav>
+    <%= render "devise/shared/error_messages", resource: resource %>
+    <div class="field">
+      <%= f.text_field :name, autofocus: true, autocomplete: "name", placeholder:'Full name', class: "form-control" %>
+    </div>
```

Comment:
- [ ] I am unable to sign up a user from the front end of the app. Despite filling in the name input, I keep getting a 'Name can't be blank' error. Kindly resolve this.

  All spaces are filled | Form is submitted
  :---:|:---:
  ![tchilo-1](https://user-images.githubusercontent.com/36542986/164293156-8cadd271-4530-49aa-8e6f-3005beef880d.png) | ![tchilo-2](https://user-images.githubusercontent.com/36542986/164293163-11addc42-c7e7-4f40-94b9-c514477b4ff5.png)

  ### SUGGESTION
  You are having this issue because you did not permit the `:name` parameter in your devise form. Kindly visit the [Strong Parameters section of the Devise documentation](https://github.com/heartcombo/devise#strong-parameters) and copy the first code snippet containing the `:configure_permitted_parameters` into your `app/controllers/application_controller.rb` file. Then edit the `keys` attribute from `[:username]` to `[:name]`